C-Plus-Plus/cp/Josephus.cpp

```diff
@@ -0,0 +1,35 @@
+/**
+Josephus problem is a famous problem.
```

Comment:
Add problem link from any competitive programming website.

Reply:
Please do add the same.